ui/components/multichain/asset-list-conversion-button/asset-list-conversion-button.js

```diff
@@ -45,6 +45,10 @@ export const AssetListConversionButton = ({ onClick, variant }) => {
       alignItems={AlignItems.center}
       borderRadius={BorderRadius.LG}
       style={{ backgroundImage, backgroundColor: color }}
+      paddingInlineStart={4}
+      paddingInlineEnd={4}
```

Comment:
Hmm! seems like the width of both the buttons are different 🤔 . 
![Screenshot 2023-10-12 at 7 29 33 PM](https://github.com/MetaMask/metamask-extension/assets/39872794/7edfd435-a853-4233-b347-bf8ac8dba56e)

![Screenshot 2023-10-12 at 7 29 16 PM](https://github.com/MetaMask/metamask-extension/assets/39872794/154f63a5-4d93-47bb-9c58-4c91a8010d5b)

Also, I think we need to decrease the paddingInline value since as per figma the recieve card should not be inline. 

![Screenshot 2023-10-12 at 7 31 03 PM](https://github.com/MetaMask/metamask-extension/assets/39872794/b7adfc48-8a5d-4441-a250-60f8649f5d9f)